docs/language/learn-ql/index.rst

```diff
@@ -34,7 +34,6 @@ If you are new to QL, start by looking at the following topics:
    introduction-to-ql
    about-ql
    beginner/ql-tutorials
-   ql-etudes/river-crossing
```

Comment:
Does this topic need an introduction too?

Reply:
This topic won't exist in the migrated help (though we can use some of the content in https://github.com/github/product-documentation/issues/2159). I've just updated the TOC so it doesn't use a broken link!